src/Components/Feed.js

```diff
@@ -0,0 +1,107 @@
+class Feed extends React.Component {
+  constructor() {
+    super();
+    this.state = { inputComment: '', feedData: [] };
+  }
+
+  componentDidMount() {
+    this.fetchFeed();
+  }
+
+  handleInput = e => {
+    this.setState({
+      [e.target.name]: e.target.value,
+    });
+  };
+
+  handleKeyDown = e => {
+    if (e.code === 'Enter') {
+      this.addComment();
+      e.target.value = '';
+    }
+  };
+
+  fetchFeed = () => {
+    fetch('http://localhost:3000/data/ParkJoonWoo/feedData.json')
+      .then(res => res.json())
+      .then(data => {
+        this.setState({ feedData: data });
+      });
+  };
+
+  addComment = () => {
+    const { feedData, inputComment } = this.state;
+
+    const input = document.querySelector('.input-comment');
+
+    if (!inputComment) {
+      return;
+    } else {
+      const newObj = {
+        id: feedData[0].comments.length + 1,
+        userName: 'Walsh',
+        comment: inputComment,
+      };
+      const addCommentToList = feedData[0].comments.concat(newObj);
+    }
+    input.value = '';
+  };
+
+  render() {
+    const { feedData } = this.state;
+    return feedData.map((data, index) => {
+      return (
+        <>
+          <div className="feeds-container">
+            <section className="feed-personal-info">
+              <span className="personal-photo"></span>
+              <span className="personal-id">{data.profileName}</span>
+            </section>
+            <section className="feed-photo-container">
+              <img alt="feed" src={data.imgSrc} className="feed-photo" />
+            </section>
+            <section className="feed-activity">
+              <article className="activity-icons">
+                <img alt="heart" src="/Images/nav/heart.svg" />
+                <i className="far fa-comment"></i>
+                <i className="far fa-paper-plane"></i>
+                <i className="far fa-bookmark"></i>
+              </article>
+              <article className="activity-good-number">
+                좋아요 {data.likeCount}개
+              </article>
+              <article className="activity-info">
+                <span className="info-personal-id">{data.profileName}</span>
+                <span className="info-personal-content">{data.content}</span>
+              </article>
+              <ul className="comment-list">
+                {feedData && feedData.length > 0 && (
+                  <CommentList commentList={feedData[index].comments} />
+                )}
+              </ul>
+              <article className="activity-time">11시간 전</article>
+            </section>
+            <section className="input-container">
+              <i className="far fa-grin"></i>
+              <input
+                type="text"
+                className="input-comment"
+                placeholder="댓글 달기..."
+                name="inputComment"
+                onChange={e => this.handleInput(e)}
```

Comment:
- onChange 와 같은 이벤트에 할당되어 있는 함수에는 자동으로 event 가 인자로 들어갑니다. 따라서 그냥 아래와 같이 작성해 주실 수 있습니다. 
```suggestion
                onChange={this.handleInput}
```